DeviceSDK/src/main/java/org/astarteplatform/devicesdk/AstarteDeviceIdHelper.java

```diff
@@ -0,0 +1,25 @@
+package org.astarteplatform.devicesdk;
+
+import java.nio.ByteBuffer;
+import java.util.Base64;
+import java.util.UUID;
+import org.astarteplatform.devicesdk.util.UUID5;
+
+public class AstarteDeviceIdHelper {
```

Comment:
`Helper` class name suffix is commonly used for instantiable classes, while `Utils` is frequently used as name suffix for static method collections.
I would go for `Utils` here.